src/e2e-test/java/io/cdap/cdap/ui/utils/Commands.java

```diff
@@ -40,10 +40,21 @@
 
 public class Commands implements CdfHelper {
 
+  // Simple Pipeline Nodes
   public static NodeInfo simpleSourceNode = new NodeInfo("BigQueryTable", "batchsource", "0");
   public static NodeInfo simpleTransformNode = new NodeInfo("Wrangler", "transform", "1");
   public static NodeInfo simpleSinkNode = new NodeInfo("BigQueryMultiTable", "batchsink", "2");
 
+  // Complex Pipeline Nodes
+  public static NodeInfo complexSourceNode1 = new NodeInfo("Spanner", "batchsource", "0");
```

Comment:
why are these commands but not constants?

Reply:
Good point but I mainly kept them here due to the simple nodes being used across different tests. Would've made this PR messy. Might move them in another refactor PR